website/docs/faqs/Accounts/delete-users.md

```diff
@@ -10,19 +10,20 @@ To delete a user in dbt Cloud, you must be an account owner or have admin privil
 
 1. From dbt Cloud, click the gear icon at the top right and select **Account Settings**.
 
-<Lightbox src="/img/docs/dbt-cloud/Navigate To Account Settings.png" title="Navigate to account settings" />
+<Lightbox src="/img/docs/dbt-cloud/Navigate To Account Settings.png" width="85%"  title="Navigate to account settings" />
 
 2. In **Account Settings**, select **Users** under **Teams**.
 3. Select the user you want to delete, then click **Edit**. 
 4. Click **Delete** in the bottom left. Click **Confirm Delete** to immediately delete the user without additional password prompts. This action cannot be undone. However, you can re-invite the user with the same information if the deletion was made in error. 
 
-<Lightbox src="/img/docs/dbt-cloud/delete_user_20221023.gif" title="Deleting a user" />
+<Lightbox src="/img/docs/dbt-cloud/delete_user_20221023.gif" width="85%" title="Deleting a user" />
 
 If you are on a **Teams** plan and you are deleting users to reduce the number of billable seats, you also need to take these steps to lower the license count:
 1. In **Account Settings**, select **Billing**. 
-2. Enter the number of developer seats you want and click **Save**.
+2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. Leaving these blank won't allow you to save your changes.
```

Comment:
```suggestion
2. Enter the number of developer seats you want and make sure you fill in all the payment details, including the **Billing Address** section. If any field is left blank, you won't be able to save your changes.
```

Reply:
this sugg is passive voice so i'm switching it to active